deepchem/models/torch_models/grover_layers.py

```diff
@@ -1,16 +1,263 @@
 """
 Copied from https://github.com/tencent-ailab/grover/blob/0421d97a5e1bd1b59d1923e3afd556afbe4ff782/grover/model/layers.py
 """
+from typing import List, Dict
 try:
     import torch
     import torch.nn as nn
 except ModuleNotFoundError:
     raise ImportError('These classes require PyTorch to be installed.')
 import numpy as np
 from scipy import stats
+from deepchem.models.torch_models.readout import GroverReadout
 from deepchem.models.torch_models.layers import SublayerConnection, PositionwiseFeedForward
 
 
+class GroverEmbedding(nn.Module):
+    """GroverEmbedding layer.
+
+    This layer is a simple wrapper over GroverTransEncoder layer for retrieving the embeddings from the GroverTransEncoder corresponding to the `atom_embedding_output_type` chosen by the user.
+
+    Parameters
+    ----------
+    edge_fdim: int
+        the dimension of additional feature for edge/bond.
+    node_fdim: int
+        the dimension of additional feature for node/atom.
+    depth: int
+        Dynamic message passing depth for use in MPNEncoder
+    undirected: bool
+        The message passing is undirected or not
+    num_mt_block: int
+        the number of message passing blocks.
+    num_head: int
+        the number of attention heads.
+    atom_embedding_output_type: str
+        the type of output aggregation after message passing.
+                                            atom_messages:      True                      False
```

Comment:
Is the formatting here messed up? It looks strange

Reply:
Fixed it